### ISSUE TITLE
📑 Feature Request: A sub-command within the CLI to list currently available models on a user's machine

### ISSUE THREAD

**Issue by DhanshreeA** [ersilia-os/ersilia#599]
### Is your feature request related to a problem? Please describe.

While trying to clean up models currently installed on my system, I have to rely on memory to recall which models I installed from `ersilia fetch` command. However I am looking for a simplistic solution like `ersilia ls | xargs ersilia delete`. If the CLI could list models currently installed on my system, I could use that information to pipe into other commands if needed.

### Describe the solution you'd like.

Implementing a sub command like `ersilia ls` or `ersilia model-ls` to list all models that were fetched using the ersilia CLI.

### Describe alternatives you've considered

I noticed that there's a sub command called `clear` but I haven't used it.

**Comment by carcablop** [comment]
Hi @DhanshreeA. A command as you propose would be very useful.
What I usually do to remember which models are the ones I have installed on my system is to list the conda environments that I have with the `conda env list` command, with this reference I can see the ones I have installed on my system and if I am going to remove them with the `ersilia delete eosxxxx` command, and there everything is deleted including the conda environment created from the model.
